frontend-public/src/components/Forms/EditReportForm.js

```diff
@@ -49,7 +48,10 @@ export const EditReportForm = (props) => (
       <Popconfirm
         placement="topRight"
         title="Are you sure?"
-        onConfirm={props.closeModal}
+        /* Cancelling here is too late because all serverside actions have been
+         * taken. Calling handleSubmit will update the props with latest data,
+         * handle document status/received_date, and close the modal */
+        onConfirm={props.handleSubmit}
```

Comment:
I think it might be time to remove the cancel button, since its not really cancelling anything at this point.
Also, you get a toast message saying that you "Successfully updated expected document" when you hit cancel, which is kinda not what you would expect to see when you hit a cancel button.

Reply:
I agree. It's pointless. If there's no requirement for it to exist then I'm happy to remove it